### PR TITLE
docs(community): add SIG introduction

### LINKED ISSUE #670: Add community section describing the SIGs
## What

Our `sig-` repositories are mostly maintained by individuals instead of actual groups forming around them.
We should further emphasize the idea behind SIGs and describe their goal 

## How

First we could add a community section describing "Special Interest Groups" in general. Additionally a brief introduction of the different SIGs we currently try to start could be made. 
Either by having every necessary and valuable information on our website, or by linking to introductions present in the `sig-` repos itself

### PR DESCRIPTION
**Pull request by SebastianBezold**
## Description

This PR adds a new menu item to the "Community" section.
This new section is introducing the idea of SIGs and contains a very basic skeleton for the already existing SIGs

Fixes #670

### CODE REVIEW

File: community/sigs/01_intro.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "What is a SIG?"
+---
+
+Eclipse Tractus-X consists of different applications, products, KITs and also hosts other supporting repositories.
+All of these repositories have a very dedicated goal and focus.
+However, there are of course cross-cutting concerns, that affect our whole project.
+
+To foster these cross-cutting needs, we decided to form "Special Interest Groups", or "SIGs" in short.
+SIGs
```
Comment (FaGru3n) [inline community/sigs/01_intro.md:10]: guess doubled.

File: community/sigs/02_sig_infra.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: "SIG Infra"
+---
+
+__Focus:__ Creating automation for various Eclipse Tractus-X topics like CI or release. 
+
+__Repository:__ [eclipse-tractusx/sig-infra](https://github.com/eclipse-tractusx/sig-infra)
+
```
Comment (FaGru3n) [inline community/sigs/02_sig_infra.md:8]:
```suggestion

__Project:__ [System Team](https://github.com/orgs/eclipse-tractusx/projects/9)

```

**Review by FaGru3n — CHANGES_REQUESTED** [review]
Thanks looks quite good, well done with little remarks.

File: community/sigs/02_sig_infra.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+## Intro
+
+SIG infra is striving to improve the developer experience in Eclipse Tractus-X, by working on different automation topics.
+We are also providing guidance on how to work with [Helm](https://helm.sh/) and support the [e2e-testing](https://github.com/eclipse-tractusx/e2e-testing)
+approach, that will be based on a umbrella Chart and Helm tests.
+
+## Dash automation
+
+A topic, every Tractus-X repo has to take care of, is legal compliance.
+One of the aspects is an up-to-date `DEPENDENCIES` file, generated by [eclipse dash-license](https://github.com/eclipse/dash-licenses).
```
Comment (hzierer) [inline community/sigs/02_sig_infra.md:18]:
```suggestion
One of the aspects is an up-to-date `DEPENDENCIES` file, generated by [eclipse dash-license](https://github.com/eclipse/dash-licenses) tool.
```

File: community/sigs/01_intro.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: "What is a SIG?"
+---
+
+Eclipse Tractus-X consists of different applications, products, KITs and also hosts other supporting repositories.
+All of these repositories have a very dedicated goal and focus.
+However, there are of course cross-cutting concerns, that affect our whole project.
```
Comment (hzierer) [inline community/sigs/01_intro.md:7]: I think "concern" is a bit dramatic here
```suggestion
However, there are of course cross-cutting topics, that affect our whole project.
```

**Review by hzierer — CHANGES_REQUESTED** [review]
nice, only two small changes

File: community/sigs/02_sig_infra.md (hunk 2 above)
Comment (SebastianBezold) [reply community/sigs/02_sig_infra.md:8]: Hmm not sure about that. I also thought about it in the beginning, but came to the conclusion, that the "System Team" focus is a bit broader than just "SIG infra". If we could split that to be more dedicated, then I would add that project later on

File: community/sigs/02_sig_infra.md (hunk 3 above)
Comment (SebastianBezold) [reply community/sigs/02_sig_infra.md:18]: I see, I missed the 's' at the end of dash-licenses. Would it be fine to add that and omit "tool". 
I think "generated by <product-name>" is enough

File: community/sigs/01_intro.md (hunk 1 above)
Comment (SebastianBezold) [reply community/sigs/01_intro.md:10]: [67ba0ea](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/676/commits/67ba0ea6255e3b2839f3461104dc98de9bb643e9)

File: community/sigs/02_sig_infra.md (hunk 3 above)
Comment (hzierer) [reply community/sigs/02_sig_infra.md:18]: yeah, that should be ok, too

File: community/sigs/02_sig_infra.md (hunk 2 above)
Comment (FaGru3n) [reply community/sigs/02_sig_infra.md:8]: ... understand your point here, but guess then we can also show them how we support and which board we use as sig-infra. Keeping that in mind also ok to start without and there could be more afterwards.

**Review by FaGru3n — DISMISSED** [review]
LGTM

File: community/sigs/01_intro.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+title: "What is a SIG?"
+---
+
+Eclipse Tractus-X consists of different applications, products, KITs and also hosts other supporting repositories.
+All of these repositories have a very dedicated goal and focus.
+However, there are of course cross-cutting topics, that affect our whole project.
+
+To foster these needs, we decided to form "Special Interest Groups", or "SIGs" in short.
+SIGs
```
Comment (tomaszbarwicki) [inline community/sigs/01_intro.md:10]: Not sure, if that is kind of leftover, orphan word? :)

**Review by tomaszbarwicki — DISMISSED** [review]

File: community/sigs/01_intro.md (hunk 5 above)
Comment (SebastianBezold) [reply community/sigs/01_intro.md]: oh that slipped through :) thanks. I think I wanted to start a new sentence, but ran out of ideas how to finish it . Removed

**Review by tomaszbarwicki — APPROVED** [review]

**Review by FaGru3n — APPROVED** [review]
LGTM

**Pull request merged**